Recrutify/Recrutify.Host/DBScripts/users_collection.js

```diff
@@ -4,71 +4,194 @@
+            "GlobalRoles": [NumberInt(4), NumberInt(3)],
             "Salt": "FDG4354GFD345542",
-            "Surname": "Lewis"
+            "Surname": "Lewis",
+            "ProjectRoles": [
+                [UUID("2beed73a-30b5-11ec-808a-fb45776a1ed3"),
+                [
+                    NumberInt(3),
+                    NumberInt(4)
+                ]],
+                [UUID("bdbc3af0-f882-4f4f-bbb8-3e87ba16b768"),
+                [
+                    NumberInt(4)
+                ]],
+                [UUID("dd49fe8f-d697-41a5-be23-7066563b85a6"),
+                [
+                    NumberInt(4)
+                ]],               
+            ]
         },
         {
             "_id": UUID("cb40d31e-7447-4ec0-bc53-ab566f3a7b2e"),
             "Email": "fredtaylor@gmail.com",
             "Name": "Fred",
             "Password": "A606B0C8919A0F6EB85240460D6772AA963669688C2CBD0D4A0B329CA238D8B0",
-            "Roles": [NumberInt(3)],
+            "GlobalRoles": [NumberInt(3)],
             "Salt": "234FGHDS234EG454",
-            "Surname": "Taylor"
+            "Surname": "Taylor",
+            "ProjectRoles": [
+                [UUID("7d85284c-30b5-11ec-95cb-230b32afd221"),
+                [
+                    NumberInt(3)
+                ]],
+                [UUID("871b1e7a-30b5-11ec-9b40-437a6473123c"),
+                [
+                    NumberInt(3)
+                ]],
+                [UUID("dd49fe8f-d697-41a5-be23-7066563b85a6"),
+                [
+                    NumberInt(3)
+                ]],
+            ]
         },
         {
             "_id": UUID("71da4db7-36ec-4a6b-bfb3-38d2462bdf2d"),
             "Email": "justinwalker@gmail.com",
             "Name": "Justin",
             "Password": "2B8CB1313018D3FAE4A5A58E2231AF82EFB6BE435574742A728AA7851615976F",
-            "Roles": [NumberInt(3)],
+            "GlobalRoles": [NumberInt(3)],
             "Salt": "65734FHGFG54634G",
-            "Surname": "Walker"
+            "Surname": "Walker",
+            "ProjectRoles": [
+                [UUID("7d85284c-30b5-11ec-95cb-230b32afd221"),
+                [
+                    NumberInt(3)
+                ]],
+                [UUID("bdbc3af0-f882-4f4f-bbb8-3e87ba16b768"),
+                [
+                    NumberInt(3)
+                ]],
+                [UUID("871b1e7a-30b5-11ec-9b40-437a6473123c"),
+                [
+                    NumberInt(3)
+                ]],
+            ]
         },
         {
             "_id": UUID("03699464-c0a6-412f-8c83-eca85b359e91"),
             "Email": "kevingarcia@gmail.com",
             "Name": "Kevin",
             "Password": "4B0BEB558891202DF7BC320DB22F8C7930BA3899D433B57A8E862CA42AF08305",
-            "Roles": [NumberInt(2)],
+            "GlobalRoles": [NumberInt(2)],
             "Salt": "AS567TYTT4535RFH",
-            "Surname": "Wils"
+            "Surname": "Wils",
+            "ProjectRoles": [
```

Comment:
you can move Surname after Name, and ProjectRoles after GlobalRoles, Salt after Password

Reply:
in all places